lib/browser/guest-view-manager.js

```diff
@@ -340,35 +340,35 @@ const isWebViewTagEnabled = function (contents) {
   return isWebViewTagEnabledCache.get(contents)
 }
 
-const handleMessage = function (channel, handler) {
-  ipcMainInternal.on(channel, (event, ...args) => {
-    if (isWebViewTagEnabled(event.sender)) {
-      handler(event, ...args)
-    } else {
-      event.returnValue = null
-    }
-  })
+const assertWebViewTagEnabled = function (event) {
+  if (!isWebViewTagEnabled(event.sender)) {
+    throw new Error('<webview> disabled')
+  }
 }
 
-handleMessage('ELECTRON_GUEST_VIEW_MANAGER_CREATE_GUEST', function (event, params, requestId) {
-  event._replyInternal(`ELECTRON_RESPONSE_${requestId}`, createGuest(event.sender, params))
+ipcMainUtils.handle('ELECTRON_GUEST_VIEW_MANAGER_CREATE_GUEST', function (event, params) {
+  assertWebViewTagEnabled(event)
+  return createGuest(event.sender, params)
 })
 
-handleMessage('ELECTRON_GUEST_VIEW_MANAGER_CREATE_GUEST_SYNC', function (event, params) {
-  event.returnValue = createGuest(event.sender, params)
+ipcMainUtils.handle('ELECTRON_GUEST_VIEW_MANAGER_CREATE_GUEST_SYNC', function (event, params) {
```

Comment:
```js
function({sender: webContents}, params) {
  <...>
}
```

It will shadow the "global" webContents we have in this file, but maybe it's not a big deal.
If it is, a different name can be used =)

Reply:
I've used `contents`